src/org/thoughtcrime/securesms/preferences/AdvancedPreferenceFragment.java

```diff
@@ -330,11 +371,15 @@ private void exportKeys() {
                           .show();
                     }
                     else {
+                        String explain = getActivity().getString(R.string.pref_managekeys_import_explain_tips);
+                        if (Build.VERSION.SDK_INT < 30) { // in older Android we always import from Download folder
+                          explain = getActivity().getString(R.string.pref_managekeys_import_explain_path, DcHelper.getImexDir().getAbsolutePath()) + "\n\n" + explain;
+                        }
                       new AlertDialog.Builder(getActivity())
```

Comment:
sth. is wrong with indentation here - `String explain ...` and `new AlertDialog ...` should have the same indentation